bash_scripts/bash_scripts.md

```diff
@@ -2,8 +2,8 @@
 module_id: bash_scripts
```

Comment:
This comment is about the empty_nano_window image which is used further down.  I think it could benefit from a border?  Just a thought.

Reply:
@pm0kjp do you have an easy way to to add a border to the image? I can think of a couple different hacks (html in LiaScript, weird ways to edit the original image), but if know how to do it in a few clicks I'd love to learn!